src/Domain/Configuration.php

```diff
@@ -211,30 +211,22 @@ private function validateAddedInsight(): \Closure
     {
         return static function ($values): bool {
             foreach ($values as $metric => $insights) {
-                if (! class_exists($metric) ||
-                    ! in_array(Metric::class, class_implements($metric), true)
+                if (!class_exists($metric) ||
+                    !in_array(Metric::class, class_implements($metric), true)
                 ) {
-                    throw new InvalidConfiguration(sprintf(
-                        'Unable to use "%s" class as metric in section add.',
-                        $metric
-                    ));
+                    throw new InvalidConfiguration(sprintf('Unable to use "%s" class as metric in section add.', $metric));
```

Comment:
We splitted it in multiple line to avoid exceeds maximum limit of 120 characters